scp.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"os"
+	"time"
+
+	scp "github.com/bramvdbogaerde/go-scp"
+	"github.com/bramvdbogaerde/go-scp/auth"
+	"golang.org/x/crypto/ssh"
+)
+
+func scpToWorker(workerURL string, source string, dest string, tarName string) error {
+	// Use SSH key authentication from the auth package
+	// we ignore the host key in this example, please change this if you use this library
+	clientConfig, err := auth.PrivateKey("ubuntu", "/home/ubuntu/.ssh/senior-design.pem", ssh.InsecureIgnoreHostKey())
```

Comment:
Make an issue for this line -- there are significant problems with it

Reply:
will do